docs/manual.rst

```diff
@@ -986,7 +987,6 @@ example will be shown for each.
   >>> LinearRing([(1,0), (1,1), (0,0)]).is_ccw
   True
 
-A ring with an undesired orientation can be reversed like this:
```

Comment:
@adishavit can you restore this line please or explain why you removed it?

Otherwise, this is a very welcome addition, thanks!

Reply:
Huh! I don’t know why this got erased. Will do.